api/libproject/devstart.rb

```diff
@@ -1039,6 +1039,7 @@ def deploy_gcs_artifacts(cmd_name, args)
   op.parse.validate
   gcc.validate
   common.run_inline %W{gsutil cp scripts/setup_notebook_cluster.sh gs://#{gcc.project}-scripts/setup_notebook_cluster.sh}
+  common.run_inline %W{gsutil cp demos/* gs://#{gcc.project}-demos/}
```

Comment:
Since you switched this to multiple notebooks/cohorts, may want to do rsync -d instead to handle removals and file renames.